src/app/client/src/app/modules/cbse-program/components/mcq-creation/mcq-creation.component.ts

```diff
@@ -174,20 +174,25 @@ export class McqCreationComponent implements OnInit, OnChanges, AfterViewInit {
     return userName;
   }
   videoDataOutput(event) {
-    this.videoShow = false;
-    this.videoSolutionData = event;
-    this.videoSolutionName = event.name;
-    this.solutionValue = event.identifier;
-    this.videoThumbnail = event.thumbnail;
-    const videoMedia: any = {};
-    videoMedia.id = event.identifier;
-    videoMedia.src = event.downloadUrl;
-    videoMedia.type = 'video';
-    videoMedia.assetId = event.identifier;
-    videoMedia.name = event.name;
-    videoMedia.thumbnail = this.videoThumbnail;
-    this.mediaArr.push(videoMedia);
-    this.showSolutionDropDown = false;
+    if(event){
+      this.videoShow = false;
+      this.videoSolutionData = event;
+      this.videoSolutionName = event.name;
+      this.solutionValue = event.identifier;
+      this.videoThumbnail = event.thumbnail;
+      const videoMedia: any = {};
+      videoMedia.id = event.identifier;
+      videoMedia.src = event.downloadUrl;
```

Comment:
@JagadishPujari Check if there is any modification required in url before adding it to the theme object.

Reply:
We are setting a relative path in manifest media array in ECML(theme object)